docs/src/shared/PlaygroundDrawer.tsx

```diff
@@ -253,30 +274,135 @@ const PlaygroundDrawer = (props: DrawerProps): JSX.Element => {
         );
     };
 
+    type PlaygroundTabPanelProps = {
```

Comment:
we usually take the type declaration out of the component scope.

Reply:
In  fact, both `PlaygroundTabPanelProps` and `PlaygroundTabPanel` can probably be taken out of `PlaygroundDrawer` to make `PlaygroundDrawer` code shorter and easier to read